cdc/entry/schema/snapshot.go

```diff
@@ -886,12 +893,86 @@ func (s *snapshot) updatePartition(tbInfo *model.TableInfo, currentTs uint64) er
 	}
 	s.currentTs = currentTs
 	// TODO: is it necessary to print changes detailly?
+	// yes
 	log.Debug("adjust partition success",
 		zap.String("schema", tbInfo.TableName.Schema),
 		zap.String("table", tbInfo.TableName.Table))
 	return nil
 }
 
+func (s *snapshot) exchangePartition(targetTable *model.TableInfo, currentTS uint64) error {
```

Comment:
could you add some comments what should be done in exchanging partition case

Reply:
yep